addons/website_event/views/event_templates.xml

```diff
@@ -11,710 +11,775 @@
+                                    </small>
+                                </div>
+                            </li>
+                        </ul> -->
                     </div>
-                </li>
-            </ul>
-        </div>
-        <div class="col-lg-4 css_noprint">
-            <div class="clearfix"/>
-            <t t-if="not event.is_online">
-                <div class="card bg-secondary" t-if="event.address_id">
-                    <h4 class="card-header">Where</h4>
-                    <div class="card-body">
-                        <t t-if="event.google_map_img()">
-                            <a t-att-href="event.google_map_link()" target="_BLANK">
-                               <img t-att-src="event.google_map_img()" width="100%%" alt="Google Maps"/>
-                            </a>
+                    <div class="col-lg-4 bg-light rounded-right shadow-sm d-print-none">
+                        <!-- Date & Time -->
+                        <div class="o_wevent_sidebar_block">
+                            <h6 class="o_wevent_sidebar_title">Date &amp; Time</h6>
+                            <span t-esc="event.with_context(tz=event.date_tz).date_begin" t-options="{'widget': 'date', 'format': 'EEEE'}"/>
+                            <h4 class="my-1" t-field="event.with_context(tz=event.date_tz).date_begin" t-options="{'widget': 'date', 'format': 'long'}" itemprop="startDate" t-att-datetime="event.date_begin"/>
+                            <t t-if="not event.is_one_day()">Start -</t>
+                            <span t-field="event.with_context(tz=event.date_tz).date_begin" t-options="{'time_only': 'true', 'format': 'short'}"/>
+                            <t t-if="event.is_one_day()">
+                                <i class="fa fa-long-arrow-right mx-1"/>
+                                <span t-field="event.with_context(tz=event.date_tz).date_end" t-options="{'time_only': 'true', 'format': 'short'}"/>
+                            </t>
+                            <t t-else="">
+                                <i class="fa fa-long-arrow-down d-block text-muted mx-3 my-2" style="font-size: 1.5rem"/>
+                                <span t-esc="event.with_context(tz=event.date_tz).date_begin" t-options="{'widget': 'date', 'format': 'EEEE'}"/>
+                                <h4 class="my-1" t-field="event.with_context(tz=event.date_tz).date_end" t-options="{'widget': 'date', 'format': 'long'}"/>
+                                <t t-if="not event.is_one_day()">End -</t>
+                                <span t-field="event.with_context(tz=event.date_tz).date_end" t-options="{'time_only': 'true', 'format': 'short'}"/>
+                            </t>
+                            <!-- Timezone -->
+                            <small t-esc="event.date_tz" class="d-block my-3 text-muted"/>
+                            <!-- TODO: Add to calendar not working -->
```

Comment:
@qsm-odoo It works on the "Registration Completed" Page. Is it possible to modify the python route to make it works on the event page too?

Reply:
I guess, I'll a look. Is this not working in your branch or is this a bug in 12.0?

Reply:
@qsm-odoo Just in my branch.